ably/ably-ui
# [WEB-2213] Add discord icon link too footer

Jira Ticket Link / Motivation
----
https://ably.atlassian.net/browse/WEB-2213



Summary of changes
----
one new discord icon file was added and displayed in the footer component



How do you manually test this?
----
Navigate to `/components/footer` in the preview app, a discord icon should be displayed in the footer


Reviewer Tasks (optional)
----




Merge/Deploy Checklist
----




Frontend Checklist
----
